command/git.go

```diff
@@ -387,6 +387,8 @@ func (c *gitFetchProject) Execute(ctx context.Context,
 				}
 			}
 		}
+		logger.Execution().Infof("Using revision '%s' for module %s", revision, moduleName)
```

Comment:
Is it possible for git's stdout to be printed instead, the way I think it works for cloning the project? https://evergreen.mongodb.com/task_log_raw/mongodb_mongo_master_enterprise_rhel_62_64_bit_compile_4686f07b4b51cb34156275c44fc1138b950f56dc_19_05_14_07_09_46/0?type=T#L11. That would be a stronger guarantee that Evergreen is cloning the right thing.